src/impls.rs

```diff
@@ -539,6 +539,22 @@ mod bstr {
         }
     }
 
+    #[cfg(fetature = "std")]
```

Comment:
This is silently passing, I think, because of the typo.

Reply:
🤦

Reply:
The fetature is dead 🦀